pages/home.vue

```diff
@@ -0,0 +1,296 @@
+
+<template>
+  <div id="page">
+    <b-container style="background-color: white">
+      <b-row>
+        <b-col>
+          <div id="drop">
+            <b-select v-model="selectedValidated" :options="optionsValidated" @change="updateFilters" />
+          </div>
+        </b-col>
+
+        <b-col>
+          <div>
+            <b-select v-model="selectedNameGroup" :options="optionsNameGroup" @change="updateFilters" />
+          </div>
+        </b-col>
+
+        <b-col>
+          <div>
+            <b-select v-model="selectedNameClient" :options="optionsNameClient" @change="updateFilters" />
+          </div>
+        </b-col>
+
+        <b-col>
+          <div v-if="$auth.user.roleName === 'ADM'">
+            <b-select v-model="selectedName" :options="optionsName" @change="updateFilters" />
+          </div>
+        </b-col>
+      </b-row>
+
+      <b-row style="float: right;">
+        <b-col>
+          <nuxt-link to="new-mission">
+            New mission
+          </nuxt-link>
+        </b-col>
+      </b-row>
+
+      <div id="content">
+        <b-row v-if="missions.length > 0">
+          <b-col v-for="mission in missions" :key="mission.id" md="auto">
+            <b-card
+              :title="mission.name"
+              :img-src="mission.image"
+              :img-alt="mission.name"
+              img-top
+              tag="article"
+              style="max-width: 18rem; height: auto !important"
+              class="w-100 mb-2"
+              border-variant="dark"
+            >
+              <b-card-text v-show="$auth.user.roleName === 'ADM'">
+                <p>{{ mission.userName }}</p>
+              </b-card-text>
+
+              <b-button variant="success" class="link-btn">
+                <nuxt-link :to="{ path: 'mission', query: { id: mission.id }}">
+                  Accéder à la fiche
+                </nuxt-link>
+              </b-button>
+            </b-card>
+          </b-col>
+        </b-row>
+        <div v-if="missions.length === 0">
+          <p v-if="selectedValidated === null && selectedNameGroup === null &&
+            selectedNameClient === null && selectedName === null"
+          >
+            You still don't have any mission.
+          </p>
+          <p v-if="selectedValidated !== null || selectedNameGroup !== null ||
+            selectedNameClient !== null || selectedName !== null"
+          >
+            No mission for the current filter.
+          </p>
+        </div>
+      </div>
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import { axiosGet } from '../services/backend.service'
```

Comment:
There is a good practice on the structure of the Vue component . The order is as follow : 
https://vuejs.org/v2/style-guide/#Component-instance-options-order-recommended